client/js/screens/Event/Event.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import {View, Image, Text, ScrollView, Button} from 'react-native';
+import ButtonDefault from '../../components/ButtonDefault';
+import RatingScore from '../../components/RatingScore';
+import {calculateRatingScore} from '../../lib/calculateRatingScore';
+import styles from './styles';
+import SafeAreaView from 'react-native-safe-area-view';
+import MapView from 'react-native-maps';
+import Reviews from '../../components/Reviews';
+
+const Event = ({navigation, eventInfo}) => {
+  return (
+    <>
+      <ScrollView>
+        <SafeAreaView>
```

Comment:
`SafeAreaView` should not be the child of the `ScrollView`. Please make the changes.

Reply:
Complete.